pkg/kubelet/apis/config/types.go

```diff
@@ -703,3 +708,13 @@ type CrashLoopBackOffConfig struct {
 	// +optional
 	MaxContainerRestartPeriod *metav1.Duration
 }
+
+// UserNamespaces contains User Namespace configurations.
+type UserNamespaces struct {
+	// SubidsPerPod is the mapping length of subids.
```

Comment:
Is this better? I'm not sure, but I don't know if "subids" means something to users. 
```suggestion
	// SubidsPerPod is the mapping length of UIDs and GIDs.
```

If you can think of something better than this suggestion, please go ahead! :)

Reply:
Should we rename `SubidsPerPod` to `IDsPerPod` too?

Reply:
Renamed